### DISCUSSION TITLE
Associating a landscape to a specific Earth location

### DISCUSSION THREAD

**Discussion by herrijw** [Stellarium/stellarium#3892] [General]
Hi!

I am trying to associate landscape to a specific Earth location. While the procedure does seem to work for 'planets' (i.e Mars etc) it doesn't seem to apply for the above.

I've activated "Use associated planet and location" before loading the landscape.
How can I associate a landscape to a specific Earth location, so that it will change according to the selected country?

landscape.ini example:
[landscape]
name = CHINA
maptex = CHINA.png
angle_rotatez = 100
[location]
planet = Earth
latitude = +26d20'00"
longitude = +110d20'00"
altitude = 90

**Comment by gzotti** [comment]
Please read chapter 7 again. You have not specified a type for your landscape. Your other entries allow me to speculate, it's either type=spherical  (7.1.3) or type=fisheye (7.1.5).

**Reply by herrijw** [reply]
gzotti thanks! I went through ch7. What correlates the landscape's image to the specific geographic location, the country/state or coordinates?

I tried the following section reg location without success:
[location]
planet = Earth
country = China
state= Beijing
latitude = +39d54'27"
longitude = +116d23'50"
altitude = 90

**Reply by gzotti** [reply]
Coordinates. Countries are irrelevant, we are all on the same planet. What happens when you select the landscape? Does the image show, but the coordinates stay in Texas?  Show your landscape panel and Location panels, maybe you have some unexpected conflicting option activated.

**Reply by herrijw** [reply]
Yes, the coordinates does seem to update when selecting the landscape, alas the landscape remain the same when selecting the coordinates.

How precise the coordinates should be vs. the city database? I am using 
[Beijing]
latitude = +39d54'27"
longitude = +116d23'50"
While
![image](https://github.com/user-attachments/assets/18fb8efb-6cf0-48dd-90b5-b82b0b75efa5)

**Reply by gzotti** [reply]
When you select coordinates or a city from the list, depending on other settings, your landscape may change to a zero-horizon with RGB color taken from the earth map. Close the location panel, and load the landscape. I asked you before to

>Show your landscape panel and Location panels, maybe you have some unexpected conflicting option activated.

Esp. switch off "take location from network".

**Reply by herrijw** [reply]
The coordinates are updating when selecting the landscape
The landscape does change when selecting a planet
The landscape does not update when selecting an Earth based location/coordinates

Location panel
![image](https://github.com/user-attachments/assets/4b773d1b-c457-4f24-ace8-0b6e48c315b3)

 landscape panel
![image](https://github.com/user-attachments/assets/614e458c-c3eb-4f93-8261-f7f8d2c79d8e)

**Reply by gzotti** [reply]
I see an old version. If something does not work as expected, first update, then write here.

**Reply by herrijw** [reply]
p.s is it possible to control the atmosphere (on/off) similar to the lightpollution string for instance? Mainly for the moon etc

**Reply by alex-w** [reply]
Shortcut `A`

**Reply by gzotti** [reply]
There is a key documented in the User Guide. (location details). Consider also declaring time zone.
